concourse/tasks/generate-test-config.yml

```diff
@@ -17,11 +17,11 @@ run:
     - -e
     - -c
     - |
-      ruby -ryaml -rjson -e "
-        config = YAML.load(STDIN)['properties']['$TEST_PROPERTIES']
-        config['admin_user'] = File.read('admin-creds/username').strip()
-        config['admin_password'] = File.read('admin-creds/password').strip()
-        puts JSON.pretty_generate(config)
-      " \
-      < cf-manifest/cf-manifest.yml \
-      > test-config/config.json
+      bosh interpolate --var-errs \
+        --var-file=admin_user=admin-creds/username \
```

Comment:
The --var-file option reads the full file content including the line-breaks which causes the authentication to fail. Fix commit: 3841b4c742e754cd7b2a236439ff718373fefd1a

Reply:
I would rather not pull the variables with --var-file way, but use `$(< file)`

Reply:
I leave this to you, how you fix it.